pulsar/internal/lookup_service.go

```diff
@@ -81,7 +81,7 @@ func (ls *lookupService) Lookup(topic string) (*LookupResult, error) {
 			return nil, err
 		}
 
-		log.WithField("topic", topic).Debugf("Got topic lookup response: %s", res)
+		log.Debugf("got topic: %s lookup response: %s", topic, res)
```

Comment:
Using the `WithField()` provides a consistent way to present the context. In this file we're already changing 3 different ways to present the topic: 
```go
log.Debugf("got topic: %s lookup response: %s", topic, res)
log.Debugf("follow redirect to broker. %v / %v - Use proxy: %v - Topic is: %s",
log.Debugf("successfully looked up topic [%s] on broker. %s / %s - Use proxy: %t",
```

`WithField()` will always attach `topic=my-topic` to the log line. If I want to grep for that topic, that makes the life much easier even in text format.

Reply:
If we really have this kind of scenario requirement, we can repackage the `WithField` method as follows:

```
func WithField(key string, value interface{}) *log.Entry {
    return log.WithField(key, value)
}
```

Reply:
I think it makes it much easier to be consistent (adding `topic=xyz` in same format for each log message) and also allows for structured logs, if desired.